breithbarbot/cropper/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Breithbarbot\\CropperBundle\\BreithbarbotCropperBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
+        "cache:clear": "symfony-cmd",
```

Comment:
This is already part of the FrameworkBundle 🤔 I am not sure if this should be added again.